app/repository/hbase/HBase.scala

```diff
@@ -3,11 +3,11 @@ package repository.hbase
 object HBase {
   val RowKeyDelimiter = "~"
   val Wildcard = "*"
-  val DefaultColumnGroup = "d"
+  val DefaultColumnFamily = "d"
```

Comment:
Method, value and variable names should be in lower camel case

Reply:
These are constants.  The Scala convention is for constants to begin with a capital, and to then follow with camelCase e.g. scala.math.Pi.

Reply:
Message received and understood